src/utilities/controllers/discon_rotor_test_controller.cpp

```diff
@@ -0,0 +1,41 @@
+#include "discon_rotor_test_controller.hpp"
+
+#include <memory>
+
+#include "discon.h"
+
+namespace openturbine::util {
+
+extern "C" {
+
+void PITCH_CONTROLLER(
+    float avrSWAP[], int* aviFAIL, const char* accINFILE, const char* avcOUTNAME, const char* avcMSG
```

Comment:
Why are we using `const char*` for `avcOUTNAME` and `avcMSG`? Shouldn't they be `char* const` to indicate a constant pointer to a mutable character array since those strings can be changed by the controller?

Reply:
Fixed.